it/resources/benchmark-in-memory-it.ini

```diff
@@ -20,7 +20,7 @@ datastore.type = in-memory
 
 
 [workloads]
-default.url = https://github.com/opensearch-project/opensearch-benchmark-workloads
+default.url = https://github.com/martin-gaievski/opensearch-benchmark-workloads
```

Comment:
Not sure the intent here was to change this in the mainline repository. To switch to your fork locally, please follow [these instructions](https://github.com/opensearch-project/opensearch-benchmark-workloads?tab=readme-ov-file#testing-the-workload).

It would be appropriate to close this PR. Thanks.

Reply:
this is not mend to be merged, it's purely to trigger integ tests CI actions for workload PR. I was following this item from the mentioned instructions:

```
Run your integration tests using GitHub actions by selecting the branch for which you committed your changes. Verify that the tests have run as expected.
```

can you please elaborate how this item can be done without creating a PR for main branch in main repo?

Reply:
@martin-gaievski This can be done in your forked repository. You can apply these benchmark.ini changes to a specific branch in your forked repository.

To run the integration tests on a specific branch: Visit the `actions` tab in your forked repository, select `integ tests` on the left. The right side of the UI should provide an option to run against a branch of OSB. You can select the branch where you applied the benchmark.ini changes. See this example
![image](https://github.com/user-attachments/assets/017958b2-93d1-4015-b6a0-3bbfdec8d1ac)

Reply:
@IanHoang, would be good to add this example [here](https://github.com/opensearch-project/opensearch-benchmark-workloads?tab=readme-ov-file#testing-the-workload).